src/datapump/jobs/geotrellis.py

```diff
@@ -813,24 +842,34 @@ def _configurations() -> List[Dict[str, Any]]:
             {
                 "Classification": "spark-defaults",
                 "Properties": {
+                    "spark.yarn.appMasterEnv.GDAL_HTTP_MAX_RETRY": "10",
                     "spark.driver.maxResultSize": "3G",
                     "spark.yarn.appMasterEnv.LD_LIBRARY_PATH": "/usr/local/miniconda/lib/:/usr/local/lib",
                     "spark.rdd.compress": "true",
                     "spark.executorEnv.LD_LIBRARY_PATH": "/usr/local/miniconda/lib/:/usr/local/lib",
-                    "spark.executor.defaultJavaOptions": "-XX:+UseParallelGC -XX:+UseParallelOldGC -XX:OnOutOfMemoryError='kill -9 %p'",
-                    "spark.shuffle.spill.compress": "true",
-                    "spark.shuffle.compress": "true",
-                    "spark.shuffle.service.enabled": "true",
-                    "spark.driver.defaultJavaOptions": "-XX:+UseParallelGC -XX:+UseParallelOldGC -XX:OnOutOfMemoryError='kill -9 %p'",
-                    "spark.dynamicAllocation.enabled": "true",
-                    "spark.yarn.appMasterEnv.AWS_REQUEST_PAYER": "requester",
                     "spark.executorEnv.AWS_REQUEST_PAYER": "requester",
-                    "spark.yarn.appMasterEnv.GDAL_HTTP_MAX_RETRY": "10",
-                    "spark.yarn.appMasterEnv.GDAL_HTTP_RETRY_DELAY": "10",
-                    "spark.yarn.appMasterEnv.GDAL_MAX_DATASET_POOL_SIZE": "450",
                     "spark.executorEnv.GDAL_HTTP_MAX_RETRY": "10",
+                    "spark.executor.defaultJavaOptions": "-XX:+UseParallelGC -XX:+UseParallelOldGC -XX:OnOutOfMemoryError\u003d\u0027kill -9 %p\u0027",
```

Comment:
I noticed the change to the defaultJavaOptions which include what looks like some Unicode escaped characters. Just checking, is this a result of MacOS's obnoxious default of trying to insert "smart" (i.e. fancy) quotes and such? Or is it intentional and required? Specifically I'm curious about the following: -XX:OnOutOfMemoryError\u003d\u0027kill -9 %p\u0027

Reply:
oh probably, this shouldn't have changed, thanks for catching